### DISCUSSION TITLE
开源之道 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#12] [Announcements]
# 开源之道 - 开源之道

https://opensourceway.community/posts/the_way_of_open_source/open_source_way/

**Comment by SarahSmj** [comment]
李先生您好，我是秒如科技的HR，是从拉勾上看到您博客找过来的，想问一下您最近有没有考虑找工作，如果您有意向，可以加微信：smj1989，具体情况我们可以进一步沟通，期待您回复
